pkg/apis/certmanager/v1/types_certificate.go

```diff
@@ -136,6 +136,11 @@ type CertificateSpec struct {
 	// denoted issuer.
 	SecretName string `json:"secretName"`
 
+	// SecretTemplate defines annotations an labels to be propagated
+	// to the Kubernetes Secret when it is created or updated.
+	// +optional
+	SecretTemplate CertificateSecretTemplate `json:"secretTemplate"`
```

Comment:
This should be optional and shouldn't be output when the user has not specified a template.

```suggestion
	SecretTemplate *CertificateSecretTemplate `json:"secretTemplate,omitempty"`
```